mrmustard/physics/bargmann.py

```diff
@@ -12,8 +12,131 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
+r"""
 This module contains functions for performing calculations on objects in the Bargmann representations.
+
+The Bargmann representation of the state can be obtained as projecting the state on to the Bargmann basis.
+
+The Bargmann basis :math:`|z\rangle_b` can be defined from the coherent state basis :math:`|z\rangle_c` 
+
+.. math:: 
+    |z\rangle_b = e^\frac{|z|^2}{2}|z\rangle_c = \sum_n \frac{z^n}{\sqrt{n!}}|n\rangle_f.
+
+Any Gaussian objects :math:`O` can be written in the Bargmann basis as a Gaussian exponential function, parametrized by a matrix :math:`A`, a vector :math:`b` and a scalar :math:`c`, which is called ``triples`` through all the documentations in MM:
+
+.. math::
+    \langle\vec{\alpha}|O\rangle = c \exp\left( \frac12 \vec{\alpha}^T A \vec{\alpha} + \vec{\alpha}^T b \right).
+
+The objects in Bargmann representation uses the :class:`~mrmustard.physics.ansatze.PolyExpAnsatz` and the information is stored in the triple (A,b,c).
+
+The expression :math:`\langle\vec{\alpha}|O\rangle` is vectorized the variables vector :math:`\vec{\alpha}`, which is different for different quantum objects. 
+As for a ``n``-mode pure Gaussian state :math:`\langle\vec{\alpha}|\psi\rangle`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_n)`.
+
+    .. code-block::
+
+        ╔═══════╗
+        ║       ║─────▶ alpha^*_0
+        ║ |psi> ║─────▶ alpha^*_1
+        ║       ║...
+        ║       ║─────▶ alpha^*_(n-1)   
+        ╚═══════╝    
+
+All the wires in ths mixed Gaussian state :math:`\langle\vec{\alpha}|\rho|\vec{\beta}\rangle`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_n, \beta_0, \beta_1,..., \beta_n)`.
```

Comment:
The schematic runs from 0 to n-1 the text below in line 44 runs from 0 to n. I think line 44 should be corrected to 0 to n-1